Description: Fix roff comment prefix generated by dtdocbook2man
 dtdocbook2man converts XConsortium SGML comments into roff comments.
 The generated output used ...\", which groff parses as an undefined
 '..' macro followed by a comment. Generate .\" instead.
Author: Reaper <JohnGrimmReaper@disroot.org>
Forwarded: not-needed

Index: cde/programs/dtdocbook/dtdocbook2man.in
===================================================================
--- cde.orig/programs/dtdocbook/dtdocbook2man.in
+++ cde/programs/dtdocbook/dtdocbook2man.in
@@ -77,7 +77,7 @@ cat > /tmp/dtm.$$.psinc <<\!
 !
 
 grep '<\!-- $''XConsortium: ' $reffile | \
-sed -e 's/<\!-- $''XConsortium:/...\\"/g' -e 's/ -->//g' >> /tmp/dtm.$$.psinc
+sed -e 's/<\!-- $''XConsortium:/.\\"/g' -e 's/ -->//g' >> /tmp/dtm.$$.psinc
 
 cat >> /tmp/dtm.$$.psinc <<\!
 .de P!
